web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/DownloadImageService.java

```diff
@@ -27,17 +27,19 @@ public class DownloadImageService {
 
     /**
      * Method contacts the IIIF server for the requested access copy image and returns it
-     * @param pidString the UUID of the file
+     * @param contentObjectRecord solr record of the file
      * @param size a string which is either "full" for full size or a pixel length like "1200"
+     * @param pidString the UUID of the file
      * @return a response entity which contains headers and content of the access copy image
      * @throws IOException
      */
-    public ResponseEntity<InputStreamResource> streamImage(String pidString, String size, String filename)
+    public ResponseEntity<InputStreamResource> streamImage(ContentObjectRecord contentObjectRecord, String size, String pidString)
             throws IOException {
 
         String url = buildURL(pidString, size);
```

Comment:
get the id from contentObjectRecord instead of passing in pidString on its own.

Reply:
ok!